bin/runtime-common/src/messages_benchmarking.rs

```diff
@@ -84,7 +79,7 @@ fn prepare_inbound_message(
 /// function.
 pub fn prepare_message_proof_from_grandpa_chain<R, FI, B>(
 	params: MessageProofParams,
-	message_destination: InteriorMultiLocation,
+	message_generator: impl Fn(usize) -> MessagePayload,
```

Comment:
@svyatonik 
what is better here `MessagePayload` or `Vec<u8>`?

Reply:
`MessagePayload` is better :) It is alias `Vec<u8>` though

Reply:
yes, I know it is the same, but wanted to be sure, thank you :)